allure-generator/src/main/javascript/helpers/attachment-uri.js

```diff
@@ -0,0 +1,11 @@
+import {SafeString} from 'handlebars/runtime';
+
+export default function attachmentUri(source) {
+  let sourceUri;
+  if (source.indexOf('http') === 0) {
```

Comment:
In two other places your test is `source.startsWith('http')`. Would it be better to be consistent here?

Reply:
Yeah definitely, I made those adjustments